src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    public double getLength() {
+        return length;
+    }
+
+    public double getWidth() {
+        return width;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: rectangle, area: "
+                + String.format("%.2f",getArea()) + " sq. units, length: "
```

Comment:
Use the `name()` method for getting the string representation of enum constants, such as color, if `getColor()` is returning an enum value. This ensures that the string representation is consistent and not affected by any overrides of the `toString()` method in the enum.